include/zephyr/bluetooth/uuid.h

```diff
@@ -1022,6 +1733,60 @@ struct bt_uuid_128 {
  */
 #define BT_UUID_RSC_FEATURE \
 	BT_UUID_DECLARE_16(BT_UUID_RSC_FEATURE_VAL)
+/**
+ *  @brief SC Control Point Characteristic UUID value
+ */
+#define BT_UUID_SC_CONTROL_POINT_VAL 0x2a55
+/**
+ *  @brief SC Control Point Characteristic
```

Comment:
What is "SC"?

Reply:
Actually no idea. This is how it is named in the document.
![image](https://user-images.githubusercontent.com/3673391/218440450-81945f0c-e504-4617-bef4-f9c573179d90.png)

Reply:
Well, it's likely related to RSC, but that doesn't really help us either...

Reply:
I'd just keep as is: The RSCS never use the explicit values of SC, so better to just use the terms from the spec :)